### DISCUSSION TITLE
renderD128 for intel hwaccel - question ?

### DISCUSSION THREAD

**Discussion by stamandr** [blakeblackshear/frigate#3115] [Ask A Question]
I have the following below in my docker-compose.yml 

```
devices:
  - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware 
```
My config file has 
``
hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format `yuv420p
``

What exactly does it mean  "**for intel hwaccel, needs to be updated for your hardware**" ? 

How would I find out if /dev/dri/renderD128 and hwaccel are actually working ?

**Comment by NickM-27** [comment]
Some cpus may use a different device for graphics. If you've added this, hardware accel args to the frigate config, and the cameras are working then hardware accel is being used.
